lib/elixir/src/elixir.erl

```diff
@@ -297,3 +297,11 @@ string_to_quoted(String, StartLine, File, Opts) when is_integer(StartLine), is_b
 
 to_binary(List) when is_list(List) -> elixir_utils:characters_to_binary(List);
 to_binary(Atom) when is_atom(Atom) -> atom_to_binary(Atom, utf8).
+
+handle_parsing_opts(File, Opts) ->
+  put(elixir_parser_file, File),
+  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of {wrap_literals_in_blocks, WrapLiteralsBool} when is_boolean(WrapLiteralsBool) ->
+      put(wrap_literals_in_blocks, WrapLiteralsBool);
```

Comment:
Maybe yes, but I think both `put`s should be close to each other and same for the `erase`s.